web/templates/settings/notification_settings.hbs

```diff
@@ -68,9 +68,9 @@
               prefix=../prefix}}
         {{/each}}
 
-        <label for="notification_sound">
+        <div for="notification_sound">
```

Comment:
I cannot understand why this is changed to `div`, since this is associated with the `select` element.

Reply:
And anyways, I think we should have this and similar changes in a separate commit with explaining the reasoning in commit message.

Reply:
Have made a separate commit with proper reasoning for this.